src/EnergyPlus/HeatRecovery.cc

```diff
@@ -2520,7 +2511,7 @@ namespace HeatRecovery {
             // Check for balanced flow condition
             this->CheckForBalancedFlow(state, local_SecInMassFlow, local_SupInMassFlow, FirstHVACIteration);
 
-            auto &perf = state.dataHeatRecovery->BalDesDehumPerfData(this->PerfDataIndex);
+            auto const &perf = state.dataHeatRecovery->BalDesDehumPerfData(this->PerfDataIndex);
```

Comment:
[src/EnergyPlus/HeatRecovery.cc:2523]:(style),[constVariable],Variable 'perf' can be declared as reference to const